src/models/login.model.js

```diff
@@ -108,12 +108,12 @@ const login = {
       });
   },
   async isTokenValid(token) {
-    return jwt.verify(token, 'secret', (err) => {
+    return jwt.verify(token, process.env.SESSION_SECRET="secret", (err) => {
       return err == undefined;
     });
   },
   generateToken(id) {
-  return jwt.sign({ id }, 'secret', {
+  return jwt.sign({ id }, process.env.SESSION_SECRET="secret", {
```

Comment:
I don't understant why you write `="secret"` ?

Reply:
It's the secret to encode the jwt token. It can be anything.

Reply:
Oh no, it's because i made wrong

Reply:
fix une boulette :laughing: